dto/src/main/java/dto/BuyerRecord.java

```diff
@@ -0,0 +1,6 @@
+package dto;
+
+import org.springframework.data.annotation.Id;
+
+public record BuyerRecord(@Id Long id, String firstName, boolean isBot, String lastName, String userName) {
```

Comment:
there are different approaches about similar data classes:

- each module owns it own version of data class
- shared library - as you did

what was the reason you choose 2nd approach?

Reply:
Well, initially it was model in module and dto as shared type in solo-standing module, to avoid code repeating.
So, i was making DTO class in this module.

But later on, i've met Record class and asked myself "why not" - so i made it and it worked instead of DTO (and it looks consice)